Description: Bump Linux-Maximum version to 5.13
  Enable 5.13 since the package contains 5.13 compat patches
Author: Colin Ian King <colin.king@canonical.com>
Origin: ubuntu
Forwarded: no
Last-Update: 2021-06-30

Index: zfs-linux-2.0.3/META
===================================================================
--- zfs-linux-2.0.3.orig/META
+++ zfs-linux-2.0.3/META
@@ -6,5 +6,5 @@ Release:      1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS
-Linux-Maximum: 5.10
+Linux-Maximum: 5.13
 Linux-Minimum: 3.10
